srcpkgs/mise/template

```diff
@@ -12,7 +12,7 @@ license="MIT"
 homepage="https://github.com/jdx/mise"
 changelog="https://github.com/jdx/mise/releases"
 distfiles="https://github.com/jdx/mise/archive/v${version}.tar.gz>${pkgname}-${version}.tar.gz"
-checksum=96684077b01178bf402873f130a040e0ad142c729e95184b9d176e3e9092299d
+checksum=38a7d5da54b8089202c7fc576a0310cc06a422e4ee2e3c5980439eb2e602495a
 
 post_patch() {
```

Comment:
`libgit2-1.8` is now packaged, this can be dropped with it

Reply:
done